src/electionguard/decryption.py

```diff
@@ -506,74 +416,30 @@ def compute_lagrange_coefficients_for_guardians(
     }
 
 
-def reconstruct_missing_tally_decryption_shares(
-    ciphertext_tally: CiphertextTally,
-    missing_guardians: DataStore[MISSING_GUARDIAN_ID, ElectionPublicKey],
-    compensated_shares: Dict[
-        MISSING_GUARDIAN_ID,
-        Dict[AVAILABLE_GUARDIAN_ID, CompensatedTallyDecryptionShare],
-    ],
-    lagrange_coefficients: Dict[
-        MISSING_GUARDIAN_ID, Dict[AVAILABLE_GUARDIAN_ID, ElementModQ]
-    ],
-) -> Optional[Dict[MISSING_GUARDIAN_ID, TallyDecryptionShare]]:
+def compute_lagrange_coefficients_for_guardian(
```

Comment:
i think we should move these lagrange coefficient helper functions to the top or the bottom of the file